src/Agent/NewRelic/Agent/Core/OpenTelemetryBridge/MeterListenerBridge.cs

```diff
@@ -165,10 +183,41 @@ public void Start()
                     // Configure metric collection and export settings
                     metricReaderOptions.PeriodicExportingMetricReaderOptions.ExportIntervalMilliseconds = DefaultOtlpExportIntervalSeconds * 1000;
                     metricReaderOptions.TemporalityPreference = MetricReaderTemporalityPreference.Delta;
-
                 });
 
-                _meterProvider = providerBuilder.Build();
+            return providerBuilder.Build();
+        }
+
+        /// <summary>
+        /// Starts the meter listener bridge to begin collecting and forwarding metrics.
+        /// Initializes the OpenTelemetry SDK, configures OTLP export, and sets up the MeterListener.
+        /// </summary>
+        public void Start()
+        {
+            if (!_configuration.OpenTelemetryMetricsEnabled)
+            {
+                _supportabilityMetricCounters?.Record(OtelBridgeSupportabilityMetric.MetricsBridgeDisabled);
+                Log.Debug("OpenTelemetry Meter Bridge is disabled via configuration.");
+                return;
+            }
+            
+            if (_connectionInfo == null)
+            {
+                Log.Debug("OpenTelemetry Meter Bridge cannot start: connection info not available.");
+                return;
+            }
+			
+			// Record that the bridge is enabled
+            _supportabilityMetricCounters?.Record(OtelBridgeSupportabilityMetric.MetricsBridgeEnabled);
```

Comment:
The spec suggests we need to record supportability metrics whether enabled or disabled, so this call (and the equivalent call when disabled) need to be moved prior to the enabled check on line 197.

We typically use a method taking a `(bool enabled)` parameter to build the enabled/disabled metric name so you can probably make a single call to record the metric.

Reply:
I also just remembered that we typically report this kind of enabled/disabled metric at startup as a "one time" metric in `AgentHealthReporter` - look for `CollectOneTimeMetrics()`. 

For consistency, we probably should move this metric reporting to `CollectOneTimeMetrics()`.